src/index.html

```diff
@@ -12,6 +12,144 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
```

Comment:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```
need to be like that